sensors:

  tray_camera_1:
    type: advanced_logical_camera
    pose:
      xyz: [-1.3, 5.8, 1.8]
      rpy: [-pi/2, pi/2, pi]

  tray_camera_2:
    type: advanced_logical_camera
    pose:
      xyz: [-1.3, -5.8, 1.8]
      rpy: [1.428898, 1.570789, 2.999697]

      
  bin_camera_1:
    type: advanced_logical_camera
    pose:
      xyz: [-2.286, 2.96, 1.8]
      rpy: [0, pi/2, pi]
    
  bin_camera_2:
    type: advanced_logical_camera
    pose:
      xyz: [-2.286, -2.96, 1.8]
      rpy: [0, pi/2, pi]
  

